### PR TITLE
Refactor play to lessons

### LINKED ISSUE #603: Play should be updated to Lessons
As per new mocks, Topic Play Tab has been updated to Topic Lessons Tab,

Screenshot 2020-01-03 at 12 11 29 PM

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Play` with `Lessons`.

### PR DESCRIPTION
**Pull request by PrarabdhGarg**
Fixes #603 

As per the new mocks, the paly tab was renamed to lessons.
Hence, the entire project was refactored, along with tests, to maintain
consistency with mocks.

## Checklist

- [ x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [ ] The PR is made from a branch that's **not** called "develop".
- [ ] The PR is made from a branch that is up-to-date with "develop".
- [x ] The PR's branch is based on "develop" and not on any other branch.
- [x ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml:165]: Uncommit this file. This must have been automatically done by android studio.

File: app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsTitleViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.topic.lessons
+
+import androidx.lifecycle.ViewModel
+
+/** [ViewModel] for title in [TopicPlayFragment]. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsTitleViewModel.kt:5]: Update this comment to 
`/** [ViewModel] for title in [TopicLessonsFragment]. */`

File: app/src/main/res/layout/topic_play_story_summary.xml [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
     <variable
```
Comment (rt4914) [inline app/src/main/res/layout/topic_play_story_summary.xml:17]: Update this file name to `topic_lessons_story_summary`

File: app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt [hunk 4]
```diff
@@ -39,7 +39,7 @@ import org.oppia.domain.topic.TEST_TOPIC_ID_0
 
 /** Tests for [TopicPlayFragment]. */
 @RunWith(AndroidJUnit4::class)
-class TopicPlayFragmentTest {
+class TopicLessonsFragmentTest {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt:42]: In this file update the function names too. They still contain `play` word in them.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks for the PR. I have suggested some changes .

Also, following files needs to be updated,

1. `topic_play_fragment` -> `topic_lessons_fragment`
2. `topic_play_story_summary` -> `topic_lessons_story_summary`
3. `topic_play_title` -> `topic_lessons_title`

Also the corresponding data binding names will also get updated.

Please update your branch name to `Fix #603: Refactor play to lessons`

**Comment by PrarabdhGarg** [comment]
Changed branch name. 
Thus refer #612 for progress

**Pull request closed without merging**